src/kixi/paloma/enrich/string.clj

```diff
@@ -1,5 +1,11 @@
 (ns kixi.paloma.enrich.string)
 
+(defn double->string [d]
+  (str (.longValue d)))
```

Comment:
does this not work with `(format ,,,)`?

Reply:
You could but it would be best (and safer to go and convert to a biginteger and format that. 
`(format "%5d" (biginteger foo))` to be honest if feels like introducing something that could go wrong. I feel safer interop'ing the raw java object in this instance.